monitoring_v3.query.Query.align raises a TypeError on execution

#### Environment details

  - OS type and version: Macos 11.1 
  - Python version: Python 3.8.5  (anaconda)
  - pip version: pip 21.0 
  - `google-cloud-monitoring` version: 2.0.0

#### Steps to reproduce
While I can execute the query
```python
from google.cloud.monitoring_v3.query import Query
query = Query(
     client=client,
     project=project_name,
     metric_type= 'storage.googleapis.com/storage/request_count'
).select_resources(
     bucket_name_prefix=bucket_name_prefix
).select_interval(
     end_time, start_time=end_time - report_interval
)
print(next(iter(query)))
```
adding an alignment aggregation fails:
```python
from google.cloud.monitoring_v3 import Aggregation
query = query.align(
    Aggregation.Aligner.ALIGN_MAX, seconds=3600
)
print(next(iter(query)))
```
and raises
```
        params = self._build_query_params(headers_only, page_size)
>       for ts in self._client.list_time_series(**params):
E       TypeError: list_time_series() got an unexpected keyword argument 'aggregation'

.../site-packages/google/cloud/monitoring_v3/query.py:446: TypeError
```

Indeed, while https://github.com/googleapis/python-monitoring/blob/8a7e2f767c71b180d708c1bf15c0a24aff32b880/google/cloud/monitoring_v3/query.py#L477

prepares an `aggregation` parameter,
https://github.com/googleapis/python-monitoring/blob/8a7e2f767c71b180d708c1bf15c0a24aff32b880/google/cloud/monitoring_v3/services/metric_service/client.py#L874
accepts no such parameter.


### Workaround
Directly use `MetricServiceClient.list_time_series`, with the aggregation specification within the `request` parameter:
```python
request = dict(
      name=project_name,
      filter=f'metric.type = "storage.googleapis.com/storage/request_count" AND resource.label.bucket_name = starts_with("{bucket_name_prefix}")',
      interval=TimeInterval(dict(
          end_time=_to_seconds_nanos(end_time),
          start_time=_to_seconds_nanos(end_time - report_interval)
      )),
      view=ListTimeSeriesRequest.TimeSeriesView.FULL,
      aggregation=Aggregation(dict(
          per_series_aligner=Aggregation.Aligner.ALIGN_SUM,
          alignment_period=dict(seconds=3600)
      ))
  )
print(next(iter(client.list_time_series(request=request))))
```

--- a/google/cloud/monitoring_v3/query.py
+++ b/google/cloud/monitoring_v3/query.py
@@ -443,7 +443,9 @@ class Query(object):
             raise ValueError("Query time interval not specified.")
 
         params = self._build_query_params(headers_only, page_size)
-        for ts in self._client.list_time_series(**params):
+
+        request = monitoring_v3.ListTimeSeriesRequest(**params)
+        for ts in self._client.list_time_series(request):
             yield ts
 
     def _build_query_params(self, headers_only=False, page_size=None):

--- a/tests/unit/test_query.py
+++ b/tests/unit/test_query.py
@@ -394,6 +394,10 @@ class TestQuery(unittest.TestCase):
         client = self._create_client(channel)
         query = self._make_one(client, PROJECT, METRIC_TYPE)
         query = query.select_interval(start_time=T0, end_time=T1)
+
+        # add a temporal alignment to test that "aggregation" query params is
+        # correctly processed
+        query = query.align(monitoring_v3.Aggregation.Aligner.ALIGN_MAX, seconds=3600)
         response = list(query.iter(headers_only=True))
 
         self.assertEqual(len(response), 2)
@@ -412,6 +416,10 @@ class TestQuery(unittest.TestCase):
             filter='metric.type = "{type}"'.format(type=METRIC_TYPE),
             interval=self._make_interval(T1, T0),
             view=monitoring_v3.ListTimeSeriesRequest.TimeSeriesView.HEADERS,
+            aggregation=monitoring_v3.Aggregation(
+                per_series_aligner=monitoring_v3.Aggregation.Aligner.ALIGN_MAX,
+                alignment_period={"seconds": 3600},
+            ),
         )
         request = channel.requests[0][1]
         self.assertEqual(request, expected_request)

